server/embed/etcd.go

```diff
@@ -211,9 +211,6 @@ func StartEtcd(inCfg *Config) (e *Etcd, err error) {
 		CorruptCheckTime:                         cfg.ExperimentalCorruptCheckTime,
 		PreVote:                                  cfg.PreVote,
 		Logger:                                   cfg.logger,
-		LoggerConfig:                             cfg.loggerConfig,
```

Comment:
Shell these properties (cfg.loggerConfig/Core/WriteSyncer) be removes as well, as they become a `no op` that can be misleading.

Reply:
Done.